package.json

```diff
@@ -28,20 +28,22 @@
     "url": "https://github.com/Polymer/tachometer/issues"
   },
   "homepage": "https://github.com/Polymer/tachometer#readme",
+  "installsOnDemand": [
```

Comment:
What's the thinking behind requiring the `installsOnDemand` field? Is it mainly communicate to a user looking at the `package.json` what it might install?

Reply:
Yeah that, and also 👋to ensure that only that set of packages are allowed to be installed on-demand👋. I didn't really flesh out the actual threat model here, so happy to remove the need for this field.